controller/controller.go

```diff
@@ -145,19 +146,27 @@ func (c *controller) updateIngresses() error {
 
 				if address := serviceMap[serviceName]; address != "" {
 					entry := IngressEntry{
-						Namespace:               ingress.Namespace,
-						Name:                    ingress.Name,
-						Host:                    rule.Host,
-						Path:                    path.Path,
-						ServiceAddress:          address,
-						ServicePort:             int32(path.Backend.ServicePort.IntValue()),
-						Allow:                   c.defaultAllow,
-						ELbScheme:               ingress.Annotations[frontendElbSchemeAnnotation],
+						Namespace:      ingress.Namespace,
+						Name:           ingress.Name,
+						Host:           rule.Host,
+						Path:           path.Path,
+						ServiceAddress: address,
+						ServicePort:    int32(path.Backend.ServicePort.IntValue()),
+						Allow:          c.defaultAllow,
+						//ELbScheme:               ingress.Annotations[frontendElbSchemeAnnotation],
 						StripPaths:              c.defaultStripPath,
 						BackendKeepAliveSeconds: c.defaultBackendTimeout,
 						CreationTimestamp:       ingress.CreationTimestamp.Time,
 					}
 
+					log.Infof("ingress: %s", ingress)
```

Comment:
There is a distinct lack of logging in this project, so 👍 for adding more, but it would be nice to know *why* this is being logged in the log message - ie. "discovered ingress: %s" or "ingress to add DNS for: %s"